lib/QueryParser.js

```diff
@@ -605,4 +609,5 @@ function countTextOccurrences(query) {
   return occurrences.length
 }
 
+// eslint-disable-next-line max-lines
```

Comment:
Was this eslint disable-rule necessary before?

Reply:
Nope, I had to add it because I added line 505 (the sorting of regexps)